src/App.tsx

```diff
@@ -1,7 +1,49 @@
 import React from 'react';
+// import classNames from 'classnames';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type Props = {
+};
+
+type State = {
+  pressedKey: string;
+};
+
+export class App extends React.Component<Props, State> {
+  state = {
+    pressedKey: '',
+  };
+
+  componentDidMount(): void {
+    document.addEventListener('keydown', (event: KeyboardEvent) => {
+      this.setState({ pressedKey: event.key });
+    });
+  }
```

Comment:
Better move this function to the separate variable, because you repeat it two times))

Reply:
and because you don't actually delete it in componentWillUnmount
https://mate.academy/learn/react/class-components#/video/3221
watch from 5:05

Reply:
ook